src/transformations/utils/build-super-calls-for-chain.ts

```diff
@@ -110,6 +109,32 @@ function isImplicitlyConstructed(contract: ContractDefinition): boolean {
 
   return (
     contract.contractKind === 'contract' &&
-    (ctor == undefined || ctor.parameters.parameters.length === 0)
+    (ctor == undefined ||
+      ctor.parameters.parameters.length === 0 )
+  );
+}
+
+function isEmpty(contract: ContractDefinition): boolean {
+  const ctor = getConstructor(contract);
+  const varInitNodes = [...findAll('VariableDeclaration', contract)].filter(
+    v => v.stateVariable && v.value && !v.constant && !hasOverride(v, 'state-variable-assignment'),
+  );
+  const modifiers = [];
+  if(ctor){
+    // we only include modifiers that don't reference base contracts
+    for (const call of ctor.modifiers) {
+      const { referencedDeclaration } = call.modifierName;
+      if (referencedDeclaration != null && !ctor.linearizedBaseContracts.includes(referencedDeclaration)) {//is not inheritance
+        modifiers.push({ call });
+      }
+    }
+  }
+
+  return (
+    contract.contractKind === 'contract' &&
+    (ctor?.body?.statements?.length == undefined &&
```

Comment:
Here we protect against ctor not including a "body" field, but we don't have the same check against ctor not having a "modifiers" field

Reply:
On the new commit the modifiers will never not exists as the intial state is an empty array